src/common/connection.ts

```diff
@@ -326,7 +326,7 @@ export class Connection extends EventEmitter {
     this._url = url
     this._config = {
       timeout: 20 * 1000,
```

Comment:
Whats the advantage of having two different timeouts, one for `connect()` and one for requests? Are there scenarios where requests will take 20 seconds to complete?

Reply:
Oh yeah. There are. Talking from our daily operations.

No, it doesn't. But there were people who asked for two different timeouts.

Reply:
👍 koresar -- thanks for the insight.

With my home internet, `connect()` has never taken more than 2 seconds for me. Requests can take a lot longer depending on the complexity of the request; some requests require rippled to do more work, I believe.